### PR TITLE
Implement new action releasedel

### PR DESCRIPTION
**Pull request by mkutlak**
Delete opsys release from db, including LOBs.

Beside OpSysRelease deletes also linked Bugzillas, Mantis Bugzillas, Repositories,
Reports, Problems and assigned Builds.

### CODE REVIEW

**Comment by mkutlak** [comment]
retest this please

File: src/pyfaf/actions/releasedel.py [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+            self.log_error("You must specify the operating system.")
+            return 1
+
+        if cmdline.opsys_release is None:
+            self.log_error("You must specify the operating system release.")
+            return 1
+
+        if cmdline.opsys not in systems:
+            self.log_error("Operating system '{0}' does not exist"
+                           .format(cmdline.opsys))
```
Comment (xsuchy) [inline src/pyfaf/actions/releasedel.py]: Can you use lazy logging?
https://stackoverflow.com/questions/4148790/lazy-logger-message-string-evaluation

File: src/pyfaf/actions/releasedel.py [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+         .delete(False))
+
+        empty_problems = get_empty_problems(db)
+        self.log_info("Empty problems found: {0}".format(len(empty_problems)))
+
+        for problem in empty_problems:
+            self.log_debug("Removing empty problem #{0}".format(problem.id))
+            db.session.delete(problem)
+
+    def _delete_package(self, pkg):
```
Comment (xsuchy) [inline src/pyfaf/actions/releasedel.py]: Can you merge it with src/pyfaf/actions/cleanup_unassigned.py:delete_package() ?

**Comment by xsuchy** [comment]
+1 LGTM

**Pull request merged**